doc/SAI_1.7.1 Release notes.md

```diff
@@ -0,0 +1,149 @@
+Reason:
+IEEE standards 802.1AE-2018 and 802.1X-2020 both state that SSCI is a property of the SA not the SC. SSCI can only be assigned once the full Live Peer List has been exchanged via EAPOL between peers, which will occur after the KaY has already created the transmit SC for the actor.
+
+`````
+ * @brief SSCI value for this Secure Association
+ *
+ * Valid when SAI_MACSEC_SC_ATTR_MACSEC_CIPHER_SUITE == SAI_MACSEC_CIPHER_SUITE_GCM_AES_XPN_128 or SAI_MACSEC_SC_ATTR_MACSEC_CIPHER_SUITE == SAI_MACSEC_CIPHER_SUITE_GCM_AES_XPN_256.
+ *
+ * @type sai_uint32_t
+ * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+`````
+SAI_MACSEC_SA_ATTR_MACSEC_SSCI,
+       Replace attribute SAI_MACSEC_SA_ATTR_ENCRYPTION_ENABLE with SAI_MACSEC_SC_ATTR_ENCRYPTION_ENABLE
+Reason:
+As seen from both the open source KaY (wpa-supplicant) and SecY (Linux MACsec driver), the encryption attribute is typically an attribute of the SC, not the SA. This alignment can be retained by shifting the SAI MACsec definition of the encryption attribute from the SA to the SC. This should allow for a cleaner, more direct translation between the SAI MACsec layer and the underlying platform driver.
+
+`````
+ * @brief True means encryption is enabled.  False means encryption is disabled.
+ *
+ * @type bool
+ * @flags CREATE_AND_SET
+ * @default true
+`````
+SAI_MACSEC_SC_ATTR_ENCRYPTION_ENABLE,
+       Replace attribute SAI_MACSEC_SC_ATTR_MACSEC_XPN64_ENABLE and SAI_MACSEC_SA_ATTR_SAK_256_BITS with SAI_MACSEC_SC_ATTR_MACSEC_CIPHER_SUITE
+Reason:
+As seen from both the open source KaY (wpa-supplicant) and SecY (Linux MACsec driver), the MACsec cipher suite is typically a single attribute of the SC. This alignment can be retained by combining the two properties of the MACsec cipher suite (SAK length and XPN enable) of the SAI MACsec definition into a single attribute of the SC. This should allow for a cleaner, more direct translation between the SAI MACsec layer and the underlying platform driver.
+
+`````
+ * @brief Cipher suite for this Secure Channel.
+ *
+ * @type sai_macsec_cipher_suite_t
+ * @flags MANDATORY_ON_CREATE | CREATE_AND_SET
+`````
+SAI_MACSEC_SC_ATTR_MACSEC_CIPHER_SUITE,
+
+The PR related to this feature is available at [PR#1156](https://github.com/opencomputeproject/SAI/pull/1156) ;[PR#1172](https://github.com/opencomputeproject/SAI/pull/1172)
+
+### Tunnel loopback packet action as resource	
+SAI_TUNNEL_ATTR_LOOPBACK_PACKET_ACTION is marked as resource and can be queried using the sai_object_type_get_availability API.
+`````
+Example
+sai_attribute_t *attr;
+sai_status_t status;
+uint64_t count;
+
+attr[0].id = SAI_TUNNEL_ATTR_TYPE;
+attr[0].value.s32 = SAI_TUNNEL_TYPE_VXLAN;
+
+attr[1].id = SAI_TUNNEL_ATTR_LOOPBACK_PACKET_ACTION;
+attr[1].value.s32 = SAI_PACKET_ACTION_DROP;
+
+status = sai_object_type_get_availability(g_switch_id, SAI_OBJECT_TYPE_TUNNEL, 2, &attr, &count);
+`````
+
+This API will return number of vxlan tunnels which can further with packet action as drop.

```

Comment:
There is a typo in the line:

This API will return number of vxlan tunnels which can **forward** with packet action as drop.

Reply:
addressed